infrastructure/iac-gip-inclusion/iam/terraform/main.tf

```
@@ -38,36 +38,20 @@ resource "scaleway_iam_policy" "terraform_ci" {
       "ProjectManager",
     ]
   }
+  # Grant full access to the terraform application for projects managed by Terraform
   rule {
-    project_ids = [data.scaleway_account_project.default.project_id]
-    permission_set_names = [
-      "DomainsDNSFullAccess",
+    project_ids = [
+      data.scaleway_account_project.default.project_id,
+      data.scaleway_account_project.emplois_cnav.project_id,
+      data.scaleway_account_project.site_institutionnel_2025.project_id
     ]
-  }
-  rule {
-    project_ids = [data.scaleway_account_project.terraform.project_id]
     permission_set_names = [
-      "ObjectStorageFullAccess",
+      "AllProductsFullAccess",
     ]
   }
+  # Although it is managed by Terraform, this project is used solely to store Terraform states
```

Je trouve plus clair de préciser que c’est intentionnel, et pas qu’on a "juste besoin de ça pour le moment". 🤷 
```suggestion
  # Although it is managed by Terraform, this project is meant to store solely Terraform states.
```